src/main/java/uk/gov/hmcts/reform/translate/controllers/DictionaryController.java

```diff
@@ -50,4 +53,17 @@ public Dictionary getDictionary() {
     public void putDictionary(@RequestBody Dictionary dictionaryRequest) {
         dictionaryService.putDictionary(dictionaryRequest);
     }
+
+    @PostMapping(path = "/translation/cy", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Translation returned successfully"),
+        @ApiResponse(responseCode = "400", description = "Bad Request (001 bad schema)"),
```

Comment:
Refere to constants when present

Reply:
I think it's easier to read and consistent across all the endpoints but happy to discuss further

Reply:
Resolved in #42